app/src/pages/header/components/AddMovieButton.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
We don't need to use it anymore:
[If you need to configure eslint](https://stackoverflow.com/questions/64646248/eslintrc-js-for-react-17-and-jsx-without-import-react)

Reply:
Thank you) Fixed